samples/graph-meeting-notification/nodejs/README.md

```diff
@@ -0,0 +1,127 @@
+---
+page_type: sample
+description: This is a sample application which demonstrates the use of online meeting subscription and sends you the notifications in chat using bot.
+products:
+- office-teams
+- office
+- office-365
+languages:
+- nodejs
+extensions:
+ contentType: samples
+ createdDate: "24-09-2022 00:16:45"
+urlFragment: officedev-microsoft-teams-samples-graph-meeting-notification-nodejs
+---
+
+# Online meeting subscription
+
+This is a sample application which demonstrates use of online meeting subscription that will post notifications when user joined/left and when meeting start/end.
```

Comment:
Create a .gif image and add it here in Readme and update the sample.json image path.